pkg/v1alpha1/handlers/events/events.go

```diff
@@ -139,5 +142,10 @@ func (h *EventHandler) ReadAll(w http.ResponseWriter, r *http.Request) {
 		totalNumberItems,
 		events,
 	}
+	// This is required in order to get the response key 'items' to be an
+	// empty JSON array instead of null when no events are returned.
+	if len(events) == 0 {
+		response.Items = make([]drivers.EiffelEvent, 0)
+	}
```

Comment:
It would perhaps be cleaner to have GetEvents return an empty slice instead of nil.

Reply:
Yes. This was poor go-knowledge on my part :)